website/style.css

```diff
@@ -1,421 +1,442 @@
 @keyframes fadeInBottom {
-    from {
-        transform: translateY(100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateY(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateY(100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInRight {
-    from {
-        transform: translateX(100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateX(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateX(100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateX(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInTop {
-    from {
-        transform: translateY(-100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateY(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateY(-100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInLeft {
-    from {
-        transform: translateX(-100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateX(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateX(-100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateX(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes slideInBottom {
-    from {
-        transform: translateY(100px);
-    }
-    to {
-        transform: translateY(0px);
-    }
+  from {
+    transform: translateY(100px);
+  }
+  to {
+    transform: translateY(0px);
+  }
 }
+
 @keyframes slideInRight {
-    from {
-        transform: translateX(100px);
-    }
-    to {
-        transform: translateX(0px);
-    }
+  from {
+    transform: translateX(100px);
+  }
+  to {
+    transform: translateX(0px);
+  }
 }
+
 @keyframes slideInTop {
-    from {
-        transform: translateY(-100px);
-    }
-    to {
-        transform: translateY(0px);
-    }
+  from {
+    transform: translateY(-100px);
+  }
+  to {
+    transform: translateY(0px);
+  }
 }
+
 @keyframes slideInLeft {
-    from {
-        transform: translateX(-100px);
-    }
-    to {
-        transform: translateX(0px);
-    }
-}
-@keyframes flexAnimation{
-    from{flex:0;}
-    to{flex:1;}
+  from {
+    transform: translateX(-100px);
+  }
+  to {
+    transform: translateX(0px);
+  }
+}
+
+@keyframes flexAnimation {
+  from {
+    flex: 0;
+  }
+  to {
+    flex: 1;
+  }
 }
 
 @keyframes fadeIn {
-    
```

Comment:
This is not used. Delete any keyframe that is not defined and is not used

Reply:
a967b37